### DISCUSSION TITLE
Previous working  Photoprism lost contact with Mariadb

### DISCUSSION THREAD

**Discussion by The1Waterman** [photoprism/photoprism#4558] [Q&A]
I have a previous working Photoprism and Maria db, which under went an Watchtower update, and since then Photoprism has lost contact with the database.  It appeared that they were on different networks, although there were listed on Portainer as being on the same, and their IP addresses confirming this.  I've had to start a new stack, which now works, but have lost all the photographic data, albums etc setup (which took me days to organise).  is there away I can re direct the new Photoprism to the old database or move left over folders from the previous install, in to the new version (previous data is in a different directory to the new stack install?)

**Comment by graciousgrey** [comment]
Our troubleshooting checklists may help with the mariadb issue:

- https://docs.photoprism.app/getting-started/troubleshooting/mariadb/#cannot-connect
- https://docs.photoprism.app/getting-started/troubleshooting/mariadb/#version-upgrade

In case you have a backup 

https://docs.photoprism.app/user-guide/backups/
https://docs.photoprism.app/user-guide/settings/advanced/#backups

you could restore this on the new installation https://docs.photoprism.app/user-guide/backups/restore/
